specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-06-01-preview/examples/ServerCreateWithPrivateEndpoint.json

```diff
@@ -0,0 +1,153 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "testrg",
+    "api-version": "2023-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "parameters": {
+      "location": "westus",
+      "sku": {
+        "tier": "GeneralPurpose",
+        "name": "Standard_D4s_v3"
+      },
+      "properties": {
+        "administratorLogin": "cloudsa",
+        "administratorLoginPassword": "password",
+        "version": "12",
+        "availabilityZone": "1",
+        "createMode": "Create",
+        "storage": {
+          "storageSizeGB": 512,
+          "autoGrow": "Disabled",
+          "tier": "P20"
+        },
+        "backup": {
+          "backupRetentionDays": 7,
+          "geoRedundantBackup": "Disabled"
+        },
+        "network": {
+          "publicNetworkAccess": "Disabled"
```

Comment:
I don't know if we need this example. Private Endpoint feature is independent of `publicNetworkAccess`.